modular_skyrat/master_files/code/modules/power/powernet.dm

```diff
@@ -0,0 +1,6 @@
+/datum/powernet/get_electrocute_damage()
+	if(avail >= 1000)
+		var/damage = clamp(20 + round(avail/25000), 20, 195) + rand(-5,5)
+		return damage * HUMAN_HEALTH_MODIFIER
+	else
+		return 0
```

Comment:
I don't think it was an oversight, it directly stated in the changelog that the amount of damage received was lowered.